test/libsolidity/semanticTests/constants/asm_numerical_underscores.sol

```diff
@@ -0,0 +1,54 @@
+contract A {
+    function b() public returns (uint i) {
+        assembly {
+            i := 3_200_000_000
+        }
+    }
+
+    function c() public returns (uint i) {
+        assembly {
+            i := 3200000000
+        }
+    }
+
+    function d() public returns (int i) {
+        assembly {
+            i := 3_200_000_000
+        }
+    }
+
+    function e() public returns (int i) {
+        assembly {
+            i := 32e8
+        }
+    }
+
+    function f() public returns (int h) {
+        assembly {
+            h := 0xBEBC2000
+        }
+    }
+
+    function g() public returns (int h) {
+        assembly {
+            h := 0xbebc2000
+        }
+    }
+
```

Comment:
```suggestion
```
Scientific notation and numeric hex literals are covered in different tests, and are unrelated to this particular one, especially seeing as it's named `asm_numerical_underscores`, and neither of these have any underscores.